### DISCUSSION TITLE
Exercises from "The Complete DX 7" book on MiniDexed

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#108] [General]
While studying the 1986 book "The Complete DX 7" to get a better understanding of how frequency modulation works using the various algorithms, I have started to write down how to do the exercises using MiniDexed.

https://github.com/probonopd/MiniDexed/wiki/Exercises

Is this useful? Or should we rather write a new tutorial from scratch?

**Comment by poetaster** [comment]
> https://github.com/probonopd/MiniDexed/wiki/Exercises
> 
> Is this useful? Or should we rather write a new tutorial from scratch?
 
I like this approach, though I think it only makes sense to continue if others in the community contribute :) I think a lot of 'tweeking' might also end up being done by 'ad-hoc' CC assignments, which I see is in the works?
